module/solutions/helper.js

```diff
@@ -68,6 +70,53 @@ module.exports = class SolutionsHelper {
     });
   }
 
+   /**
+    * Update solution document.
+    * @method
+    * @name updateSolutionDocument
+    * @param {Object} query - query to find document
+    * @param {Object} updateObject - fields to update
+    * @returns {String} - message.
+    */
+
+   static updateSolutionDocument(query= {}, updateObject= {}) {
```

Comment:
@priyanka-TL  Why new helper functions ?? We can use existing update helper functions :pensive:

Reply:
@aman-tunerlabs we don't have a helper function for update solutions. we are using findOneAndUpdate